### PR TITLE
add the basic FAS theme

### PR DESCRIPTION
**Pull request by ryanlerch**
This adds a new theme, FAS for use with the Fedora Account System

Signed-off-by: Ryan Lerch <rlerch@redhat.com>

### CODE REVIEW

**Comment by ryanlerch** [comment]
Added another commit which adds a macro to the theme allowing a theme to specify an element to use for changing a group's details.

Resolves: #46 #47

File: securitas/themes/fas/templates/main.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+{% endblock %}
+
+
+{# an optional macro defining an element to show for editing group details #}
+{# typically, it used to link to a ticket tracker for an admin to edit a group #}
+{% macro edit_group_details() %}
+    {% if current_user_is_sponsor and sponsor_form %}
+        <div class="col-auto ml-auto">
+            <a class="btn btn-sm btn-outline-primary" href="https://pagure.io/fedora-infrastructure/new_issue" title="file a Fedora Infra ticket to change the details or sponsors of this group">
+                request change of details
```
Comment (abompard) [inline securitas/themes/fas/templates/main.html]: Maybe capitalize this?

**Review by abompard — CHANGES_REQUESTED** [review]
One more thing: the "Add user" input on the group page is back to having joined borders with the users list.

File: securitas/themes/fas/templates/main.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    <script src="{{ url_for('theme.static', filename='js/jquery/jquery-1.10.2.min.js') }}"></script>
+    <script src="{{ url_for('theme.static', filename='js/fedora-bootstrap/fedora-bootstrap.min.js') }}"></script>
+    {{ super() }}
+{% endblock %}
+
+
+{# an optional macro defining an element to show for editing group details #}
+{# typically, it used to link to a ticket tracker for an admin to edit a group #}
+{% macro edit_group_details() %}
+    {% if current_user_is_sponsor and sponsor_form %}
```
[the remark this answers is not in the document]
Comment (ryanlerch) [reply securitas/themes/fas/templates/main.html:75]: i was just using the same logic we used for actually showing the add user box to sponsors of a group:

https://github.com/fedora-infra/securitas/blob/ea49362a00688a4dd15cd19cb8cf74be2885d43a/securitas/templates/group.html#L44

**Comment by ryanlerch** [comment]
> One more thing: the "Add user" input on the group page is back to having joined borders with the users list.

fixed this one too

**Review by abompard — APPROVED** [review]
Looks good, thanks!

**Pull request merged**